SE80_GIT testDDRExt_General_0 DDRInteractiveCommandException: Unrecognized type 'KeyHashTableClassEntry'

Failure link
------------

From [an internal build](http://vmfarm.rtp.raleigh.ibm.com/job_output.php?id=125415271):
```
java version "1.8.0_491"
Java(TM) SE Runtime Environment (build 8.0.9.0 - pxl6480sr9-20260327_01(SR9))
IBM J9 VM (build 2.9, JRE 1.8.0 Linux ppc64le-64-Bit Compressed References 20260416_115986 (JIT enabled, AOT enabled)
OpenJ9   - a7234b698ce
OMR      - 4a4463e19
IBM      - 764a44b)
JCL - 20260318_01 based on Oracle jdk8u491-b09
```

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------
```
Running test testDDRExt_General_0 ...
===============================================
testDDRExt_General_0 Start Time: Thu Apr 16 22:21:55 2026 Epoch Time (ms): 1776392515790
variation: NoOptions
JVM_OPTIONS:  

     [java] [AutoRun] [INFO] Starting testWalkJ9Pool(j9vm.test.ddrext.junit.TestDDRExtensionGeneral)
     [java] 1050624021627263506543659216372801047104857632768331201310720-120484549480262144000000130736256214528130736256214568130736256214608130736256214648300001307363594053120130736254365136524288096324525632451110013107213073633905740802048151316323200100112345429261516384409611111111110111111110083650000192512-108160010000000105-11748000276801307362562146881307362562147280-13000128011101024102560800523275600058182430464669066553637610409681624[DDRExtTesterBase] [INFO] Validation started for : '!walkj9pool 0x000076E768127520'
     [java] [DDRExtTesterBase] [INFO] Validation passed for : '!walkj9pool 0x000076E768127520'
     [java] 32125409681620[DDRExtTesterBase] [INFO] Validation started for : '!walkj9pool 0x000076E7681263E0'
     [java] [DDRExtTesterBase] [INFO] Validation passed for : '!walkj9pool 0x000076E7681263E0'
     [java] 112354096816214748365018[DDRExtTesterBase] [ERROR] '!walkj9hashtable 0x000076E7681375E0 KeyHashTableClassEntry' output contains common failure key : DDRInteractiveCommandException
     [java] [DDRExtTesterBase] [ERROR] '!walkj9hashtable 0x000076E7681375E0 KeyHashTableClassEntry' output :
     [java] Problem running command:
     [java] com.ibm.j9ddr.tools.ddrinteractive.DDRInteractiveCommandException: Unrecognized type 'KeyHashTableClassEntry'
     [java] 	at com.ibm.j9ddr.vm29.tools.ddrinteractive.commands.WalkJ9HashTableCommand.run(WalkJ9HashTableCommand.java:111)
     [java] 	at com.ibm.j9ddr.tools.ddrinteractive.Context.tryCommand(Context.java:220)
     [java] 	at com.ibm.j9ddr.tools.ddrinteractive.Context.execute(Context.java:196)
     [java] 	at com.ibm.j9ddr.tools.ddrinteractive.Context.execute(Context.java:176)
     [java] 	at com.ibm.j9ddr.tools.ddrinteractive.DDRInteractive.execute(DDRInteractive.java:361)
     [java] 	at j9vm.test.ddrext.DDRExtTesterBase.exec(DDRExtTesterBase.java:97)
     [java] 	at j9vm.test.ddrext.junit.TestDDRExtensionGeneral.testWalkJ9HashTableforClassHashTable(TestDDRExtensionGeneral.java:674)
     [java] 	at j9vm.test.ddrext.junit.TestDDRExtensionGeneral.testWalkJ9HashTablesForCLHashTables(TestDDRExtensionGeneral.java:647)
     [java] 	at j9vm.test.ddrext.junit.TestDDRExtensionGeneral.testWalkJ9HashTable(TestDDRExtensionGeneral.java:619)
```

It seems related to 
* https://github.com/eclipse-openj9/openj9/pull/23526

FYI @tomal-majumder @keithc-ca

--- a/runtime/ddr/vmddrstructs.properties
+++ b/runtime/ddr/vmddrstructs.properties
@@ -52,6 +52,7 @@ omrhashtable.h,\
 omrhookable.h,\
 j9modifiers_api.h,\
 j9nonbuilder.h,\
+KeyHashTable.h,\
 j9javaaccessflags.h,\
 j9nongenerated.h,\
 omrthread_generated.h,\

--- a/runtime/vm/KeyHashTable.c
+++ b/runtime/vm/KeyHashTable.c
@@ -27,6 +27,7 @@
 #include "j9port_generated.h"
 #endif /* defined(J9VM_OPT_SNAPSHOTS) */
 #include "j9protos.h"
+#include "KeyHashTable.h"
 #include "vm_internal.h"
 #include "ut_j9vm.h"
 
@@ -36,13 +37,6 @@
 
 #define ROUNDING_GRANULARITY    4
 #define ROUNDED_BYTE_AMOUNT(number)  (((number) + (ROUNDING_GRANULARITY - 1)) & ~(UDATA)(ROUNDING_GRANULARITY - 1))
-
-typedef union KeyHashTableClassEntry {
-	UDATA tag;
-	J9Class *ramClass;
-	J9PackageIDTableEntry packageID;
-	J9UTF8 *className;
-} KeyHashTableClassEntry;
 
 typedef struct KeyHashTableClassQueryEntry {
 	KeyHashTableClassEntry entry;

--- a/runtime/vm/KeyHashTable.h
+++ b/runtime/vm/KeyHashTable.h
@@ -1,0 +1,35 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2026
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ *******************************************************************************/
+
+#ifndef KEYHHASHTABLE_H_
+#define KEYHHASHTABLE_H_
+
+#include "j9nonbuilder.h"
+
+typedef union KeyHashTableClassEntry {
+	UDATA tag;
+	struct J9Class *ramClass;
+	J9PackageIDTableEntry packageID;
+	struct J9UTF8 *className;
+} KeyHashTableClassEntry;
+
+#endif /* KEYHHASHTABLE_H_ */

